src/thunderbots/software/ai/hl/stp/tactic/tactic.h

```diff
@@ -121,13 +121,12 @@ class Tactic
      *
      * @param yield The coroutine push_type for the Tactic
      *
-     * @return A unique pointer to the next Intent that should be run for the Tactic.
+     * @yield A unique pointer to the next Intent that should be run for the Tactic.
      * If the Tactic is done, an empty/null unique pointer is returned. The very first
      * time this function is called, a null pointer will be returned (this does not
      * signify the Tactic is done).
```

Comment:
Similar comment to above about `@yield`. Same goes for other instances of it in this PR.

Reply:
fixed